### PR TITLE
Add one book and one course in English

### PR DESCRIPTION
**Pull request by Raibows**
## What does this PR do?
Add resource(s)

### Description
The first one is a book for high-dimensional data analysis.
~~The second one is a course for computational data analysis.~~

### Why is this valuable (or not)?
It is useful for someone to learn data analysis especially for theory learning.

### How do we know it's really free?
The book is on gthub.io and the author states the book is free for download to view.
~~The course is on Berkeley website, 100% free.~~

### For book lists, is it a book? For course lists, is it a course? etc.
The first commit is a book.
~~The second commit is a course.~~

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -336,6 +336,7 @@
 
 * [Advanced Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/advanceddataminingwithweka/)
 * [Applied Data Science with Python](https://cognitiveclass.ai/learn/data-science-with-python) - CognitiveClass.ai
+* [Berkeley's EECS 208: Computational Principles for High-Dimensional Data Analysis](https://pages.github.berkeley.edu/UCB-EECS208/course_site) (University of California, Berkeley)
```
Comment (eshellman) [inline courses/free-courses-en.md:339]: Don't invent titles! Use the title given by the resource.
Please add author names where appropriate.
This appears to be time-limited. we don't accept those.

Comment (Raibows) [reply courses/free-courses-en.md]: Hi, thanks for you advice. I have reverted the commit of Berkeley's EECS 208 course.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by Raibows** [comment]
@eshellman Hi, anything else I need to do to help merge this PR? I see this PR is labeld with ``waiting for changes``.

**Comment by eshellman** [comment]
sorry I missed the update among over a thousand PRs.
Thanks, and welcome to free-programming-books!

**Pull request merged**